backend/serializers.py

```diff
@@ -0,0 +1,30 @@
+from django.contrib.auth.models import User
+
+from rest_framework import serializers
+from rest_auth.registration.serializers import RegisterSerializer
+
+from allauth.account.adapter import get_adapter
+from allauth.account.utils import setup_user_email
+
+class AuthRegisterSerializer(RegisterSerializer):
+
+    def get_cleaned_data(self):
+        return {
+            'username': self.validated_data.get('username', ''),
+            'password1': self.validated_data.get('password1', ''),
+            'email': self.validated_data.get('email', ''),
+            'is_active': False
```

Comment:
can we put a comment here saying why we are setting this field to false? something about the admin approval requirement

Reply:
added